go/pkg/zen/session.go

```
@@ -57,6 +57,20 @@ func (s *Session) AuthorizedWorkspaceID() string {
 	return s.workspaceID
 }
 
+func (s *Session) UserAgent() string {
+	return s.r.UserAgent()
+}
+
+func (s *Session) Location() string {
+	location := s.r.Header.Get("True-Client-IP")
```

golangci-lint (1.64.8)

non-canonical header "True-Client-IP", instead use: "True-Client-Ip"

(canonicalheader)